.travis.yml

```diff
@@ -9,9 +9,10 @@ julia:
   - 1
   - nightly
 
-allow_failures:
-  - os: windows
-  - julia: nightly
+matrix:
```

Comment:
Trying jobs instead of matrix in anticipation of a documentation job: 
https://juliadocs.github.io/Documenter.jl/stable/man/hosting/#Travis-CI-1

Reply:
Perfect, thanks!